README.md

```diff
@@ -68,7 +68,7 @@ Now your templates are available to the client generator, and you can write outp
 ## But how do I modify this?
 The `TwilioGoGenerator.java` has comments in it--lots of comments.  There is no good substitute for reading the code more, though.  See how the `TwilioGoGenerator` implements `CodegenConfig`. That class has the signature of all values that can be overridden.
 
-You can also step through TwilioGoGenerator.java in a debugger.  Just debug the JUnit test in DebugCodegenLauncher. That runs the command line tool and lets you inspect what the code is doing.  
+You can also step through TwilioGoGenerator.java in a debugger.  Just debug the JUnit test in DebugCodegenLauncher. That runs the command line tool and lets you inspect what the code is doing.
```

Comment:
Has anyone actually tried doing this?

Reply:
I think this is left over from when we migrated Sam’s original POC. I’ll test this out and verify this works. If it doesn’t I’ll go ahead and remove it.

Reply:
I've done this and it works.